rollup.config.mjs

```diff
@@ -2,14 +2,28 @@ import { nodeResolve } from '@rollup/plugin-node-resolve';
 import typescript from '@rollup/plugin-typescript';
 import postcss from 'rollup-plugin-postcss';
 
+/**
+ * @type {import('rollup').RollupOptions}
+ */
 export default {
   input: 'src/index.ts',
-  output: {
-    dir: 'lib',
-    format: 'es',
-    preserveModules: true,
-    preserveModulesRoot: 'src',
-  },
+  output: [
+    {
+      dir: 'lib',
+      format: 'es',
+      preserveModules: true,
+      preserveModulesRoot: 'src',
+      sourcemap: true,
+    },
+    {
+      dir: 'lib',
+      format: 'cjs',
+      preserveModules: true,
+      preserveModulesRoot: 'src',
+      sourcemap: true,
+      entryFileNames: '[name].cjs',
```

Comment:
This builds the cjs files using `.cjs` extension into the same directories, no longer requiring separate `es` and `cjs` directories... I prefer it this way but we can always go back to separate directories

Reply:
Either way seems reasonable, so let's go with your preference.